From f9b609a3fe878258f6348bf8dc5a19853d7b718e Mon Sep 17 00:00:00 2001
From: Alon Levy <alevy@redhat.com>
Date: Mon, 3 May 2010 12:55:57 -0300
Subject: [PATCH 03/15] spice-vmc: add braces to single line if's

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1272891368-32706-3-git-send-email-alevy@redhat.com>
Patchwork-id: 8960
O-Subject: [RHEL-6 PATCH 02/13] spice-vmc: add braces to single line if's
Bugzilla: 576488
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

From: alon.levy <alevy@redhat.com>

BZ: 576488
---
 hw/spice-vmc.c |   12 +++++++++---
 1 files changed, 9 insertions(+), 3 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/spice-vmc.c |   12 +++++++++---
 1 files changed, 9 insertions(+), 3 deletions(-)

diff --git a/hw/spice-vmc.c b/hw/spice-vmc.c
index 60385f7..555395c 100644
--- a/hw/spice-vmc.c
+++ b/hw/spice-vmc.c
@@ -100,8 +100,10 @@ static int spice_virtual_channel_interface_write(
 {
     SpiceVirtualChannel *svc = container_of(port, SpiceVirtualChannel, interface);
     ssize_t written = virtio_serial_write(&svc->port, buf, len);
-    if (written != len)
+
+    if (written != len) {
         printf("WARNING: %s short write. %lu of %d\n", __func__, written, len);
+    }
 
    /* TODO:
     * we always claim the write worked. Reason: otherwise interface gives up
@@ -177,7 +179,9 @@ static void spice_virtual_channel_vm_change_state_handler(
 
     if (running) {
         svc->running = true;
-        if (svc->plug) svc->plug->wakeup(svc->plug);
+        if (svc->plug) {
+            svc->plug->wakeup(svc->plug);
+        }
     } else {
         svc->running = false;
     }
@@ -227,7 +231,9 @@ static void spice_virtual_channel_have_data(
     }
     svc->guest_out_ring.bytes += bytes_read;
     // wakeup spice
-    if (svc->plug) svc->plug->wakeup(svc->plug);
+    if (svc->plug) {
+        svc->plug->wakeup(svc->plug);
+    }
     return;
 }
 
-- 
1.7.0.3

